test/lib/slots.test.ts

```diff
@@ -0,0 +1,56 @@
+import getSlots from '@lib/slots';
+import {it, expect} from '@jest/globals';
+import MockDate from 'mockdate';
```

Comment:
Codacy has a fix for the issue: [Replace `'mockdate'` with `"mockdate"`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7689306)
```suggestion
import MockDate from "mockdate";
```